src/components/oculus-touch-controls.js

```diff
@@ -220,25 +224,25 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     if (!data.model) { return; }
     // Set the controller display model based on the data passed in.
     this.displayModel = CONTROLLER_PROPERTIES[data.controllerType] || CONTROLLER_PROPERTIES[CONTROLLER_DEFAULT];
-    // If the developer is asking for auto-detection, see if the displayName can be retrieved to identify the specific unit.
-    // This only works for WebVR currently.
+    // If the developer is asking for auto-detection, use the retrieved displayName to identify the specific unit.
     if (data.controllerType === 'auto') {
       var trackedControlsSystem = this.el.sceneEl.systems['tracked-controls-webvr'];
-      // WebVR
       if (trackedControlsSystem && trackedControlsSystem.vrDisplay) {
+        // WebVR
         var displayName = trackedControlsSystem.vrDisplay.displayName;
-        // The Oculus Quest uses the updated generation 2 inside-out tracked controllers so update the displayModel.
         if (/^Oculus Quest$/.test(displayName)) {
           this.displayModel = CONTROLLER_PROPERTIES['oculus-touch-v2'];
         }
-      } else { // WebXR
+      } else {
+        // WebXR
```

Comment:
this should move to the line above as it was before

Reply:
I found having one to the side, and one above, to make the code consuing to read. Given that they are symmetrical comments describing symmetrical code blocks, this was a way to make them consistent.

The other way to make them consistent would be to have them both to the side

```js
if { // webvr
// ...
} else { // webxr
// ...
```

but that feels messier. It also feels like a comment outside of the entire `if` block describes the entire `if/else` chain, where as a comment _inside_ the `if` block describes what is inside that block (which is more accurate).

As someone who was working on this code and having to grok those comments, are you willing to reconsider this reposition? I personally really feel it is an improvement. If you insist, though, I can put them back I suppose.